myhoard/web_server.py

```diff
@@ -75,7 +75,7 @@ async def backup_list(self, _request):
             }
             with self.controller.lock:
                 if self.controller.state["backups_fetched_at"]:
-                    response["backups"] = self.controller.state["backups"]
+                    response["backups"] = list(self.controller.state["backups"].values())
```

Comment:
**comment** left as a list here, I don't see a reason for changing the response type